fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/ExternalResourceCacheEntry.java

```diff
@@ -36,7 +36,7 @@
  * @since 2017-09-18
  */
 public class ExternalResourceCacheEntry extends BinaryCacheEntry {
```

Comment:
Not sure if its worth pursuing right now, but I think we could probably remove the CacheEntryFactory at this point since it is exclusively being used for fixity checks inside two FedoraBinary implementations, which now have the internal knowledge about whether they are external or not.